sdks/python/apache_beam/examples/complete/game/leader_board.py

```diff
@@ -217,7 +217,7 @@ def expand(self, pcoll):
         # processing of late data.
         | 'LeaderboardTeamFixedWindows' >> beam.WindowInto(
             beam.window.FixedWindows(self.team_window_duration),
-            trigger=trigger.AfterWatermark(trigger.AfterCount(10),
+            trigger=trigger.AfterWatermark(trigger.AfterCount(20),
```

Comment:
This is changing the example, not the test. I think we want to demonstrate both early and late triggers here.

Is there a way to fix this in the test?

Reply:
I see, then the behavior that this PR is trying to fix is actually expected, I think we should change the test to verify the number of rows is larger than 10 instead of equal to 10. As it's the pipeline which generates the big query rows.

Reply:
Thanks, closing this PR.